### PR TITLE
Added BestReplyMessages

### PR DESCRIPTION
**Pull request by Sameer-MSFT**
Added BestReplyMessages, please publish on July 1

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: yammer/manage-security-and-compliance/export-yammer-enterprise-data.md [hunk 1]
```diff
@@ -132,6 +132,7 @@ To find this data for an individual user, click the Yammer settings icon ![Yamme
...
 |**LikedMessages.csv** <br/> | For any message liked by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the ids for attachments, and creation and deletion information. A list of polls you created will also be provided. For announcements, includes the title of the announcement. <br/> |
 |**Messages.csv** <br/> | For any message sent or modified by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the ids for attachments, and creation and deletion information. A list of polls you created will also be provided. For announcements, includes the title of the announcement. <br/> |
+|**BestReplyMessages.csv** <br/> | For any message marked as best reply by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the ids for attachments, and creation and deletion information. <br/>|
```
Comment (nenonix) [inline yammer/manage-security-and-compliance/export-yammer-enterprise-data.md]:
```suggestion
|**BestReplyMessages.csv** <br/> | For any message marked as best reply by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the IDs for attachments, creation and deletion information. <br/>|
```

**Comment by mypil** [comment]
@Sameer-MSFT - we need your kind assistance to merge the new commit for the copy/edit review made by nenonix or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by Sameer-MSFT** [comment]
Thanks for the comments, looks good.

**Comment by Sameer-MSFT** [comment]
Thanks Joy. I’ve just commented on the copy – it looks good.

Thanks,
Sameer

**Pull request merged**